src/tests/ftest/cart/test_group_np_common.h

```diff
@@ -54,21 +73,37 @@ struct test_t test_g = { .t_hold_time = 0,
 			 .t_srv_ctx_num = 1,
 			 .t_roomno = 1082 };
 
-#define CRT_ISEQ_TEST_PING_CHECK /* input fields */		 \
+/* input fields */
+#define CRT_ISEQ_TEST_PING_CHECK				 \
 	((uint32_t)		(age)			CRT_VAR) \
 	((uint32_t)		(days)			CRT_VAR) \
 	((d_string_t)		(name)			CRT_VAR) \
 	((bool)			(bool_val)		CRT_VAR)
 
-#define CRT_OSEQ_TEST_PING_CHECK /* output fields */		 \
+/* output fields */
+#define CRT_OSEQ_TEST_PING_CHECK				 \
 	((int32_t)		(ret)			CRT_VAR) \
 	((uint32_t)		(room_no)		CRT_VAR) \
 	((uint32_t)		(bool_val)		CRT_VAR)
 
 CRT_RPC_DECLARE(test_ping_check,
 		CRT_ISEQ_TEST_PING_CHECK, CRT_OSEQ_TEST_PING_CHECK)
 CRT_RPC_DEFINE(test_ping_check,
-		CRT_ISEQ_TEST_PING_CHECK, CRT_OSEQ_TEST_PING_CHECK)
+	       CRT_ISEQ_TEST_PING_CHECK, CRT_OSEQ_TEST_PING_CHECK)
+
+/* input fields */
+#define CRT_ISEQ_TEST_SWIM_STATUS				 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
Ignoring: `checkpatch.pl` can't seem to analyze complex macros properly.